bin/ufds-reconcile-data

```diff
@@ -59,33 +61,33 @@ function setup_vars {
     #From util.sh
     download_metadata
 
-    UFDS_ADMIN_IP=127.0.0.1
-    UFDS_LDAP_ROOT_DN=$(json -f ${METADATA} ufds_ldap_root_dn)
-    UFDS_LDAP_ROOT_PW=$(json -f ${METADATA} ufds_ldap_root_pw)
+    UFDS_LDAP_ROOT_DN="$(json -f "${METADATA:?}" ufds_ldap_root_dn)"
```

Comment:
So we trainwreck if ufds_ldap_root_dn is not set now.  I'm very cool with this, FTR.